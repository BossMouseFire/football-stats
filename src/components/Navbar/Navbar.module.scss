.navbar{
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: left;
  background: #4e7934;
  width: 100vw;
  height: 60px;
  color: white;
  padding: 0 30px;

  img{
    width: 25px;
    height: 25px;
  }
  span{
    font-size: 24px;
    margin-left: 10px;
    a {
      color: white;
    }
  }
}
