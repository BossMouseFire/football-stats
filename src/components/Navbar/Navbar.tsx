import React from 'react';
import styles from './Navbar.module.scss';
import Ball from '../../assets/imgs/soccer-ball.svg';

export const Navbar = () => {
    return (
        <div className={styles.navbar}>
            <img src={Ball} alt='ball' />
            <span>
                <a id='link-main-page' href={'/'}>
                    FootSTAT
                </a>
            </span>
        </div>
    );
};
