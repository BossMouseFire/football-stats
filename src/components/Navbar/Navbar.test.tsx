import { Navbar } from './Navbar';
import { renderToJson } from '../../test/helpers/renderToJson';

it('Navbar renders correctly', () => {
    const navbar = renderToJson(<Navbar />);

    expect(navbar).toMatchSnapshot();
});

export {};
