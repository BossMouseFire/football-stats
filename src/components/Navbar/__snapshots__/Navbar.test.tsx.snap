// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  className="navbar"
>
  <img
    alt="ball"
    src="soccer-ball.svg"
  />
  <span>
    <a
      href="/"
      id="link-main-page"
    >
      FootSTAT
    </a>
  </span>
</div>
`;
